terraform/app.tf

```diff
@@ -33,9 +33,9 @@ locals {
       EncryptionKeys__1                            = local.infrastructure_secrets.ENCRYPTION_KEY1,
       SigningKeys__0                               = local.infrastructure_secrets.SIGNING_KEY0,
       SigningKeys__1                               = local.infrastructure_secrets.SIGNING_KEY1,
-      Notify__ApiKey                               = local.infrastructure_secrets.NOTIFY_API_KEY,
+      Notify__ApiKey                               = local.infrastructure_secrets.NOTIFY_ID_API_KEY,
```

Comment:
Keys have changed so we can roll this out more easily, since we're also moving to a new Notify service.